### PR TITLE
GitHub redirect

### PR DESCRIPTION
**Pull request by carolynyoo**
github3

### CODE REVIEW

File: server/app/index.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ app.use(function (req, res, next) {
 
 });
 
+app.get('/github', function (req, res, next) {
+  res.redirect('https://github.com/carolynyoo/stack_store');
```
Comment (darksim735) [inline server/app/index.js:33]: :)

File: package.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
     "lodash": "^3.3.0",
     "mocha-mongoose": "^1.0.3",
     "mongodb": "^1.4.33",
-    "mongoose": "^3.8.23",
+    "mongoose": "3.8.23",
```
Comment (carolynyoo) [inline package.json:43]: sry really wanted to change this...the unstable version of mongoose thing is killing me

**Comment by darksim735** [comment]
:hamburger:

**Comment by itsaliu90** [comment]
:ok:

**Comment by carolynyoo** [comment]
hold on merges til randy's merge conflicts

**Pull request merged**